### PR TITLE
fix: Update the quickstart to match the new, updated quickstart

### PR DESCRIPTION
**Pull request by mieubrisse**
I also started updating the CircleCI to use YAML anchors because I discovered we were duplicating things and the data-package hadn't been added, but I didn't go all the way

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -11,6 +11,20 @@ install_and_start_kurtosis: &install_and_start_kurtosis
       kurtosis cluster set docker
       kurtosis engine start
 
+# Run quickstart (used in two places)
+run_quickstart: &run_quickstart
+  run: |
+    kurtosis run ./quickstart '[{"first_name":"Kevin", "last_name": "Bacon"}, {"first_name":"Steve", "last_name":"Buscemi"}]'
```
Comment (h4ck3rk3y) [inline .circleci/config.yml:17]: i think the failure is coming from the sibling package "run ." workflow not being supported :-( 

@Peeeekay fysa

Comment (Peeeekay) [reply .circleci/config.yml:17]: oh yeah that's not supported; are we calling quickstart some other example package?

Comment (Peeeekay) [reply .circleci/config.yml:17]: yepp; that's not supported currently at least while running locally. Running it remotely should still work. In the pipeline, you could do something like `kurtosis run github.com/kurtosis-tech/awesome-kurtosis/quickstart@{CIRCLE_BRANCH}` should work.

Comment (h4ck3rk3y) [reply .circleci/config.yml:17]: Echo'ing that @victorcolombo ran into this too

**Comment by h4ck3rk3y** [comment]
@mieubrisse I am merging the main.star changes to main and using @Peeeekay's CI work around for now. I haven't transferred the YAML anchors to reduce scope

https://github.com/kurtosis-tech/awesome-kurtosis/pull/94

**Comment by h4ck3rk3y** [comment]
Adding YAML anchors to #87  and closing this

**Pull request closed without merging**